linode/rancher-ui-driver-linode
# Fix dependency vulnerabilities alert

## 📝 Description

Fixed a couple of critical vulnerabilities by 
- Bumping babel/core and babel/traverse to 7.23.2
- Bump trim-newlines to 4.0.1
- Bump send to 0.19.0

Should address https://github.com/linode/rancher-ui-driver-linode/security/dependabot/42, https://github.com/linode/rancher-ui-driver-linode/security/dependabot/46, and https://github.com/linode/rancher-ui-driver-linode/security/dependabot/14.

## ✔️ How to Test

1. Run `npm install` against main branch and see a couple of vulnerabilities found.
2. Pull this branch and run `npm install`.
3. No vulnerabilities found.
4. Run `npm run build` and no error raised.
